grpc/jwt/jwt.go

```diff
@@ -98,8 +89,8 @@ func JWTokenSource(secretKey []byte, pem bool, clientID uuid.UUID) (oauth2.Token
 	}
 
 	t := jwt.New()
-	_ = t.Set(jwt.IssuerKey, clientID.String())
-	_ = t.Set(jwt.SubjectKey, clientID.String())
+	_ = t.Set(jwt.IssuerKey, clientID)
+	_ = t.Set(jwt.SubjectKey, clientID)
```

Comment:
Why do we set both Issuer and Subject to clientId? that does not make sense to me

Reply:
Because for AppAgent Issuer of token is the same as Subject.